.vscode/lit.code-snippets

```diff
@@ -1,30 +1,36 @@
 {
-	"Create Lit Component": {
-		"prefix": "lit new",
+	"UmbNewLitElement": {
+		"prefix": "new umb element",
 		"scope": "typescript",
 		"body": [
-			"import { UmbTextStyles } from '@umbraco-ui/backoffice/style';",
-			"import { css, html } from 'lit';",
-			"import { customElement } from 'lit/decorators.js';",
+			"import { css, customElement, html } from '@umbraco-cms/backoffice/external/lit';",
 			"import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';",
+			"import { UmbTextStyles } from '@umbraco-cms/backoffice/style';",
 			"",
-			"@customElement('umb-${TM_FILENAME_BASE/(.*)\\..+$/$1/}')",
-			"export class Umb${TM_FILENAME_BASE/(.*)\\..+$/${1:/pascalcase}/}Element extends UmbLitElement {",
-			"\trender() {",
-			"\t\treturn html`${0:umb-${TM_FILENAME_BASE/(.*)\\..+$/$1/}}`;",
+			"const elementName = 'umb-${TM_FILENAME_BASE/(.*)\\..+$/$1/}';",
```

Comment:
We have this in a few places, but was it ever a convention? It prevents one of the custom eslint plugins from running, which checks for correct element names, I think?

Reply:
I didn't know this was causing an issue with an eslint plugin. I can revert this.

Reply:
I think it's nicer this way, as you only have to change an element's name in one place. Maybe we can update the eslint rule accordingly. But let us keep it here!

Reply:
Oh, I'd already removed this code before the merge. Let's discuss with the rest of the frontend team at our next meeting.